src/Node/Express/Middleware/Static.purs

```diff
@@ -0,0 +1,16 @@
+module Node.Express.Middleware.Static
+  ( static
+  ) where
+
+import Control.Monad.Eff.Class (liftEff)
+import Data.Function.Uncurried (Fn3, runFn3)
+import Data.Unit (Unit)
+import Node.Express.Handler (Handler, HandlerM(..))
+import Node.Express.Types (Request, Response, ExpressM)
+import Prelude (($))
+
+foreign import _static :: ∀eff. String -> Fn3 Request Response (ExpressM eff Unit) (ExpressM eff Unit)
```

Comment:
Could you please use `forall` instead of Unicode equivalent? It will be more consistent with existing code.